src/main/resources/i18n/aerospace/common.properties

```diff
@@ -190,6 +190,13 @@ CLIENT_CONNECTION=Client Connection (incoming)
 SERVER_CONNECTION=Server Connection (outgoing)
 DESTINATION_QUEUE=Destination queue:
 REPLY_TO_QUEUE=Reply to queue:
+PROVIDER_URL_LABEL=Provider URL:
+INITIAL_CONTEXT_CLASS_LABEL=Initial context class:
+CONNECTION_FACTORY_LABEL=Connection factory:
+USER_NAME_LABEL=Username:
+PASSWORD_LABEL=Password:
+CONFIGURATION_DETAILS=Configuration details
+DEFAULT_CONFIGURATION=DEFAULT CONFIGURATION
```

Comment:
I'll change this to be "ActiveMQ JMS" which is consistent with configuration caption for queue browser settings.
Later if we add Kafka and RabbitMQ, this title here need to be dynamic.
![image](https://user-images.githubusercontent.com/35082985/199565486-cfd50f15-5d96-4dd6-8926-a0839608499c.png)